instrumentation/jetty-httpclient/jetty-httpclient-9.2/library/src/test/java/io/opentelemetry/instrumentation/jetty/httpclient/v9_2/JettyHttpClient9LibraryTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jetty.httpclient.v9_2;
+
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.AbstractHttpClientTest;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientInstrumentationExtension;
+import java.util.Arrays;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class JettyHttpClient9LibraryTest extends AbstractJettyClient9Test {
+
+  @RegisterExtension
+  static final InstrumentationExtension extension = HttpClientInstrumentationExtension.forLibrary();
+
+  @Override
+  protected HttpClient createStandardClient() {
+    return JettyClientTelemetry.builder(testing.getOpenTelemetry())
+        .setCapturedRequestHeaders(Arrays.asList(AbstractHttpClientTest.TEST_REQUEST_HEADER))
+        .setCapturedResponseHeaders(Arrays.asList(AbstractHttpClientTest.TEST_RESPONSE_HEADER))
```

Comment:
Nit: you can use `Collections.singletonList()` here
```suggestion
        .setCapturedRequestHeaders(singletonList(AbstractHttpClientTest.TEST_REQUEST_HEADER))
        .setCapturedResponseHeaders(singletonList(AbstractHttpClientTest.TEST_RESPONSE_HEADER))
```